scalikejdbc-config/src/main/scala/scalikejdbc/config/TypesafeConfigReader.scala

```diff
@@ -97,24 +97,28 @@ trait TypesafeConfigReader extends NoEnvPrefix with LogSupport { self: TypesafeC
   }
 
   def loadGlobalSettings(): Unit = {
-    for {
-      globalConfig <- readConfig(config, envPrefix + "scalikejdbc.global")
-      logConfig <- readConfig(globalConfig, "loggingSQLAndTime")
-    } {
-      val enabled = readBoolean(logConfig, "enabled").getOrElse(false)
-      if (enabled) {
-        val default = LoggingSQLAndTimeSettings()
-        GlobalSettings.loggingSQLAndTime = LoggingSQLAndTimeSettings(
-          enabled = enabled,
-          singleLineMode = readBoolean(logConfig, "singleLineMode").getOrElse(default.singleLineMode),
-          printUnprocessedStackTrace = readBoolean(logConfig, "printUnprocessedStackTrace").getOrElse(default.printUnprocessedStackTrace),
-          stackTraceDepth = readInt(logConfig, "stackTraceDepth").getOrElse(default.stackTraceDepth),
-          logLevel = readString(logConfig, "logLevel").map(v => Symbol(v)).getOrElse(default.logLevel),
-          warningEnabled = readBoolean(logConfig, "warningEnabled").getOrElse(default.warningEnabled),
-          warningThresholdMillis = readLong(logConfig, "warningThresholdMillis").getOrElse(default.warningThresholdMillis),
-          warningLogLevel = readString(logConfig, "warningLogLevel").map(v => Symbol(v)).getOrElse(default.warningLogLevel))
-      } else {
-        GlobalSettings.loggingSQLAndTime = LoggingSQLAndTimeSettings(enabled = false)
+    readConfig(config, envPrefix + "scalikejdbc.global").foreach { globalConfig =>
+      GlobalSettings.loggingSQLErrors = readBoolean(globalConfig, "loggingSQLErrors").getOrElse(GlobalSettings.loggingSQLErrors)
+      GlobalSettings.loggingConnections = readBoolean(globalConfig, "loggingConnections").getOrElse(GlobalSettings.loggingConnections)
+
+      for {
+        logConfig <- readConfig(globalConfig, "loggingSQLAndTime")
+      } {
+        val enabled = readBoolean(logConfig, "enabled").getOrElse(GlobalSettings.loggingSQLAndTime.enabled)
```

Comment:
I changed this line:
https://github.com/scalikejdbc/scalikejdbc/pull/838/files#diff-06ac3f5fc49b361d02c1cdba1c26b28bL104
`getOrElse(false)` to `getOrElse(GlobalSettings.loggingSQLAndTime.enabled)`.

Is it a breaking change?(should I revert?)
This line is called when config contains `scalikejdbc.global.loggingSQLAndTime` but it doesn't have `enabled`.
I think the `GlobalSettings` default value is better than `false` fixed.

Reply:
fair enough